src/tvheadend/Subscription.h

```diff
@@ -0,0 +1,141 @@
+#pragma once
+
+/*
+ *      Copyright (C) 2005-2015 Team Kodi
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, write to
+ *  the Free Software Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+ *  http://www.gnu.org/copyleft/gpl.html
+ *
+ */
+
+#include "platform/threads/mutex.h"
+
+extern "C"
+{
+#include "libhts/htsmsg.h"
+}
+
+/* streaming uses a weight of 100 by default on the tvh side  */
+/* lowest configurable streaming weight in tvh is 50          */
+/* predictive tuning should be lower to avoid conflicts       */
+/* weight 0 means that tvh will use the weight of it's config */
+enum eSubscriptionWeight {
```

Comment:
Could you move this, `eSubscriptionState` and `PACKET_QUEUE_DEPT` inside the `tvheadend` namespace?